web_qr/controllers/qr.py

```diff
@@ -0,0 +1,17 @@
+import werkzeug
+from odoo import http
+from odoo.http import request
+
+
+class Home(http.Controller):
+
+    @http.route('/report/qr', type='http', auth="public")
+    def report_qr(self, value, box_size=3, border=3, **kwargs):
```

Comment:
According to https://github.com/lincolnloop/python-qrcode/blob/master/README.rst the minimum border should be 3.